src/components/Search/utils.ts

```diff
@@ -0,0 +1,67 @@
+import regions, { RegionType, County, getStateFips } from 'common/regions';
+import { sortBy, partition } from 'lodash';
+import { countyFipsToZips } from 'components/MapSelectors/datasets';
+import { stateColor, countyColor } from 'common/colors';
+
+// Todo (Chelsi): replace the many anys
+
+function getCountyZips(countyFips: string): string[] {
+  return countyFipsToZips[countyFips];
+}
```

Comment:
We should probably add the zip codes to the `County` class and add them on `common/regions/regions_data` to keep things centralized

Reply:
k can add that in